diff --git a/Makefile b/Makefile
index 8070266..dd20da0 100644
--- a/Makefile
+++ b/Makefile
@@ -31,7 +31,7 @@ dist/$(VERSION):
 	# $ go tool nm dist/v$(VERSION)/var | grep VERSION
 	#  8b0780 D _/Users/me/path/to/variant/cli/version.VERSION
 	#  6dff9c R _/Users/me/path/to/variant/cli/version.VERSION.str
-	go build -ldflags "-X '_$(shell pwd)/cli/version.VERSION=$(VERSION)'" -o dist/$(VERSION)/var .
+	go build -ldflags "-X 'github.com/mumoshu/variant/cli/version.VERSION=$(VERSION)'" -o dist/$(VERSION)/var .
 
 release: dist/$(VERSION)
 	ghr -u $(GITHUB_USER) -r $(GITHUB_REPO) -c master --prerelease v$(VERSION) dist/$(VERSION)
diff --git a/api/flow/provided_inputs.go b/api/flow/provided_inputs.go
index 0ace074..090e27f 100644
--- a/api/flow/provided_inputs.go
+++ b/api/flow/provided_inputs.go
@@ -1,10 +1,10 @@
 package flow
 
 import (
-	"../../util/maputil"
 	"fmt"
 	log "github.com/Sirupsen/logrus"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/util/maputil"
 )
 
 type ProvidedInputs struct {
diff --git a/api/step/execution_context.go b/api/step/execution_context.go
index a321735..a7c4001 100644
--- a/api/step/execution_context.go
+++ b/api/step/execution_context.go
@@ -1,7 +1,7 @@
 package step
 
 import (
-	"../flow"
+	"github.com/mumoshu/variant/api/flow"
 	"text/template"
 )
 
diff --git a/api/step/step_config.go b/api/step/step_config.go
index 2672a78..c6b676a 100644
--- a/api/step/step_config.go
+++ b/api/step/step_config.go
@@ -1,9 +1,9 @@
 package step
 
 import (
-	"../../util/maputil"
 	"github.com/Sirupsen/logrus"
 	log "github.com/Sirupsen/logrus"
+	"github.com/mumoshu/variant/util/maputil"
 	"reflect"
 )
 
diff --git a/cmd/env.go b/cmd/env.go
index 39a28e5..3d819f0 100644
--- a/cmd/env.go
+++ b/cmd/env.go
@@ -19,8 +19,8 @@ import (
 
 	"github.com/spf13/cobra"
 
-	env "../cli/env"
-	subcommands "./env"
+	env "github.com/mumoshu/variant/cli/env"
+	subcommands "github.com/mumoshu/variant/cmd/env"
 )
 
 var EnvCmd = &cobra.Command{
diff --git a/cmd/env/set.go b/cmd/env/set.go
index 272fa26..c43e2f5 100644
--- a/cmd/env/set.go
+++ b/cmd/env/set.go
@@ -19,7 +19,7 @@ import (
 
 	"github.com/spf13/cobra"
 
-	env "../../cli/env"
+	env "github.com/mumoshu/variant/cli/env"
 )
 
 var SetCmd = &cobra.Command{
diff --git a/cmd/version.go b/cmd/version.go
index 147275c..0282629 100644
--- a/cmd/version.go
+++ b/cmd/version.go
@@ -18,7 +18,7 @@ import (
 	"github.com/Sirupsen/logrus"
 	"github.com/spf13/cobra"
 
-	version "../cli/version"
+	version "github.com/mumoshu/variant/cli/version"
 )
 
 func VersionCmd(log *logrus.Logger) *cobra.Command {
diff --git a/engine/application.go b/engine/application.go
index 926a964..0ba43b5 100644
--- a/engine/application.go
+++ b/engine/application.go
@@ -11,9 +11,9 @@ import (
 	bunyan "github.com/mumoshu/logrus-bunyan-formatter"
 	"github.com/spf13/viper"
 
-	"../api/flow"
-	"../api/step"
-	"../util/maputil"
+	"github.com/mumoshu/variant/api/flow"
+	"github.com/mumoshu/variant/api/step"
+	"github.com/mumoshu/variant/util/maputil"
 	"reflect"
 )
 
diff --git a/engine/bound_flow.go b/engine/bound_flow.go
index 81726dd..e486aed 100644
--- a/engine/bound_flow.go
+++ b/engine/bound_flow.go
@@ -9,9 +9,9 @@ import (
 	"fmt"
 	"github.com/juju/errors"
 
-	"../util/maputil"
+	"github.com/mumoshu/variant/util/maputil"
 
-	"../api/step"
+	"github.com/mumoshu/variant/api/step"
 )
 
 type BoundFlow struct {
diff --git a/engine/cobra.go b/engine/cobra.go
index 186677d..a99049a 100644
--- a/engine/cobra.go
+++ b/engine/cobra.go
@@ -1,10 +1,10 @@
 package engine
 
 import (
-	flowApi "../api/flow"
 	"fmt"
 	log "github.com/Sirupsen/logrus"
 	"github.com/juju/errors"
+	flowApi "github.com/mumoshu/variant/api/flow"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 	"os"
diff --git a/engine/config.go b/engine/config.go
index 3da478e..4d3d2e2 100644
--- a/engine/config.go
+++ b/engine/config.go
@@ -8,8 +8,8 @@ import (
 	"github.com/juju/errors"
 	"github.com/mitchellh/mapstructure"
 
-	"../api/step"
-	"../util/maputil"
+	"github.com/mumoshu/variant/api/step"
+	"github.com/mumoshu/variant/util/maputil"
 )
 
 type FlowConfig struct {
diff --git a/engine/engine.go b/engine/engine.go
index 44180a6..bb64874 100644
--- a/engine/engine.go
+++ b/engine/engine.go
@@ -7,8 +7,8 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 
-	"../api/step"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/api/step"
 	"gopkg.in/yaml.v2"
 )
 
diff --git a/engine/execution_context_impl.go b/engine/execution_context_impl.go
index 9f8f151..98d6fe1 100644
--- a/engine/execution_context_impl.go
+++ b/engine/execution_context_impl.go
@@ -1,8 +1,8 @@
 package engine
 
 import (
-	"../api/flow"
-	"../api/step"
+	"github.com/mumoshu/variant/api/flow"
+	"github.com/mumoshu/variant/api/step"
 	"text/template"
 )
 
diff --git a/engine/flow.go b/engine/flow.go
index f84bd8b..0983434 100644
--- a/engine/flow.go
+++ b/engine/flow.go
@@ -1,7 +1,7 @@
 package engine
 
 import (
-	"../api/step"
+	"github.com/mumoshu/variant/api/step"
 	"github.com/spf13/cobra"
 )
 
diff --git a/engine/flow_key.go b/engine/flow_key.go
index c044402..c489185 100644
--- a/engine/flow_key.go
+++ b/engine/flow_key.go
@@ -1,8 +1,8 @@
 package engine
 
 import (
-	"../api/step"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/api/step"
 	"strings"
 )
 
diff --git a/engine/flow_registry.go b/engine/flow_registry.go
index 07ff64c..14c957b 100644
--- a/engine/flow_registry.go
+++ b/engine/flow_registry.go
@@ -1,8 +1,8 @@
 package engine
 
 import (
-	"../api/step"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/api/step"
 )
 
 type FlowRegistry struct {
diff --git a/main.go b/main.go
index 049e558..77491dc 100644
--- a/main.go
+++ b/main.go
@@ -11,12 +11,12 @@ import (
 	"github.com/juju/errors"
 	"github.com/spf13/viper"
 
-	"./cli/env"
-	"./cmd"
-	"./engine"
-	"./steps"
-	"./util/envutil"
-	"./util/fileutil"
+	"github.com/mumoshu/variant/cli/env"
+	"github.com/mumoshu/variant/cmd"
+	"github.com/mumoshu/variant/engine"
+	"github.com/mumoshu/variant/steps"
+	"github.com/mumoshu/variant/util/envutil"
+	"github.com/mumoshu/variant/util/fileutil"
 )
 
 func init() {
diff --git a/steps/flow_step.go b/steps/flow_step.go
index 99ac7e8..922fd6c 100644
--- a/steps/flow_step.go
+++ b/steps/flow_step.go
@@ -1,9 +1,9 @@
 package steps
 
 import (
-	"../api/flow"
-	"../api/step"
 	"fmt"
+	"github.com/mumoshu/variant/api/flow"
+	"github.com/mumoshu/variant/api/step"
 )
 
 type FlowStepLoader struct{}
diff --git a/steps/if_step.go b/steps/if_step.go
index 2384786..27bff35 100644
--- a/steps/if_step.go
+++ b/steps/if_step.go
@@ -1,10 +1,10 @@
 package steps
 
 import (
-	"../api/step"
-	"../util/maputil"
 	"fmt"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/api/step"
+	"github.com/mumoshu/variant/util/maputil"
 	"log"
 	"reflect"
 )
diff --git a/steps/or_step.go b/steps/or_step.go
index b7b93c2..e2f5ec6 100644
--- a/steps/or_step.go
+++ b/steps/or_step.go
@@ -1,10 +1,10 @@
 package steps
 
 import (
-	"../api/step"
-	"../util/maputil"
 	"fmt"
 	"github.com/juju/errors"
+	"github.com/mumoshu/variant/api/step"
+	"github.com/mumoshu/variant/util/maputil"
 	"log"
 	"reflect"
 )
diff --git a/steps/script_step.go b/steps/script_step.go
index 2ff9d43..39752c8 100644
--- a/steps/script_step.go
+++ b/steps/script_step.go
@@ -13,7 +13,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/juju/errors"
 
-	"../api/step"
+	"github.com/mumoshu/variant/api/step"
 )
 
 type ScriptStepLoader struct{}
